great_expectations/validator/validation_graph.py

```diff
@@ -411,31 +401,26 @@ def get_exception_info(
         self,
         metric_info: Dict[
```

Comment:
Is this an `_AbortedMetricsInfoDict`?

Reply:
No, the key is a string not a `_MetricKey`

Reply:
That's true for the return type, but not this one correct?

Reply:
yup, you're right.